### ISSUE TITLE
os has no method tmpdir

### ISSUE THREAD

**Issue by mikkoh** [tldr-pages/tldr#35]
Currently when I do:

```
tldr lsof
```

I get an error:

```
var FOLDER = path.join(os.tmpdir(), 'tldr-cache');
                          ^
TypeError: Object #<Object> has no method 'tmpdir'
    at Object.<anonymous> (/usr/local/lib/node_modules/tldr/lib/cache.js:8:27)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/tldr/lib/request.js:2:15)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
```
